implementer-guide/setup/how-to-use-azure-ad-to-login-to-trs.md

```diff
@@ -26,121 +26,89 @@ In this article we’ll assume that Telerik Report Server is installed and acces
 
 ### 1.  Setup the Azure AD Application
 
-We can use either the default directory or creating a new entry in the active directory list – depending on the users and rights you’ve already defined, you might want to use some existing directory. For simplicity, we’ll create a new entry called *ActiveDir* so your directories list should look like this:
+In our Azure portal, create an **Enterprise** application and marked it as a "**Non-gallery app**". It should look like this:
 
-![AzureAD - default directory](../../images/report-server-images/HowToLoginUsingADFS/01. Active Directory - Default.png)
+![AzureAD - application setup](../../images/report-server-images/HowToLoginUsingADFS/azure_ad_step1.png)
 
-In this directory we have to register two applications. The first one is the Telerik Report Server, which will act as a Web API server and respond to the requests made from the second application we’re about to register – the Standalone Report Designer.
 
--   Enter the directory you’ve created and add a new application using the buttons at the bottom. Select “Add an application my organization is developing” from the panel that should appear:
+### 2.  Inspect IDs:
 
-![](../../images/report-server-images/HowToLoginUsingADFS/02. Add Application P1.png)
 
--   On the next page enter “**TelerikReportServer**” the name of the application and select “Web application and/or Web API” from the radio buttons list below:
+After creating the app, inspect its IDs in the **Overview** page:
 
-![](../../images/report-server-images/HowToLoginUsingADFS/03. Add Application P2.png)
+![AzureAD - inspect-id](../../images/report-server-images/HowToLoginUsingADFS/azure_ad_step2.png)
 
--   On the third page enter the **Sign-on URL** and the **App ID URI** – both could be set to the actual URL of Telerik Report Server:
 
-![](../../images/report-server-images/HowToLoginUsingADFS/04. Add Application P3.png)
+### 3.  Set up the Single Sign On settings:
 
-When you click the checkmark button, the application will be added to your directory and you can preview its details using the CONFIGURE menu entry on the top.
+In the "Single Sign On" page we chose SAML protocol and configured the following properties:
```

Comment:
In the "Single Sign On" page, choose SAML and configure the following properties: